### PR TITLE
only update column_indices once.

### PR DESCRIPTION
**Pull request by emkornfield**

### CODE REVIEW

File: tensorflow_io/bigquery/kernels/bigquery_lib.h [hunk 1]
```diff
@@ -334,7 +334,7 @@ class BigQueryReaderAvroDatasetIterator
     const avro::GenericRecord &record =
         this->datum_->template value<avro::GenericRecord>();
 
-    if (this->column_indices_.size() == 0) {
+    if (this->column_indices_.size() == 0 && !this->column_indices_.empty()) {
```
Comment (emkornfield) [inline tensorflow_io/bigquery/kernels/bigquery_lib.h:337]: wrong place.

**Comment by vlasenkoalexey** [comment]
Looks good

**Comment by emkornfield** [comment]
actually I think I reversed the condition.

**Comment by emkornfield** [comment]
ok, think it is fixed now.

**Pull request merged**